components/eamxx/src/share/grid/remap/coarsening_remapper.cpp

```diff
@@ -388,6 +569,12 @@ local_mat_vec (const Field& x, const Field& y) const
     {
       auto x_view = x.get_view<const Pack**>();
       auto y_view = y.get_view<      Pack**>();
+      view_2d<Real> mask_view("",x_view.extent_int(0),x_view.extent_int(1));
+      if (mask != NULL) {
+        mask_view = mask->get_view<Real**>();
+      } else {
+        Kokkos::deep_copy(mask_view,1.0);
+      }
       const int dim1 = PackInfo::num_packs(src_layout.dim(1));
       auto policy = ESU::get_default_team_policy(nrows,dim1);
       Kokkos::parallel_for(policy,
```

Comment:
Note to self, the mask_view doesn't appear to be used here.  Need to investigate.